### ISSUE TITLE
Is it possible to use T5 gguf versions?

### ISSUE THREAD

**Issue by schen18** [DenOfEquity/PixArt-Sigma-for-webUI#5]
Is it possible to skip the hardcoded unquantized T5 and instead use the Forge UI vae/encoder dropdown to specify the gguf versions in https://huggingface.co/city96/t5-v1_1-xxl-encoder-gguf/  ?

**Comment by DenOfEquity** [comment]
No, they didn't exist when this extension was written so using alternate text encoders was never a consideration. Using the Forge functions to handle the text encoding would be an interesting thing to try, but it would be a very low priority experiment: performance is fine, and any such changes wouldn't benefit A1111 or reForge or old Forge.
